### ISSUE TITLE
Configurable Background Color

### ISSUE THREAD

**Issue by fenriskiba** [fenriskiba/drawcsify#3]
Right now, the background color on the diagram is defaulted to white. That should be configurable in case anyone wants to use something else to match their Docsify theme.

**Comment by rantoniuk** [comment]
The background of the SVG should be set to transparent via [SVG export settings](https://www.drawio.com/doc/faq/export-to-svg) - then it should be themable via https://jhildenbiddle.github.io/docsify-themeable/#/customization?id=plugin-styles.

**Comment by fenriskiba** [comment]
Yes, the SVG is exported as transparent, but I've added a [solid white background to the DIV around the SVG](https://github.com/fenriskiba/drawcsify/blob/main/drawcsify.js#L34) because a dark theme background (like one of the options I use from themeable) can cause color contrast issues with the diagram.

This issue would just be making that configurable. It shouldn't be hard, it's just not something I've spent the time on yet.
